src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,39 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({sender, body, timeStamp, onLike, onUnlike}) => {
+  const [buttonText, setButtonText] = useState('🤍');
+
+  const onLikeClick = () => {
+    if (buttonText === '🤍') {
+      setButtonText('❤️');
+      onLike();
+    } else {
+      setButtonText('🤍');
+      onUnlike();
+    }
+  };
```

Comment:
What I'd like to see I'd like to see is an event handler written in app and this event handler would be passed down through props from `App` to `ChatLog` and finally down to `ChatEntry`.  

`ChatEntry` could use this event handler passed down as a prop by registering it to the `<button>` element. 

This is similar to how we implemented the pet-a-cat button in the `Cat` component in flasky-front-end. Have a look at [App.jsx here from lines 44-55](https://github.com/Ada-C22/flasky-front-end/blob/db8e1afd6d2e4cadf627841adcb1baec55544857/src/App.jsx#L44-L55) to see how we write the event handler. Also have a look at [Cat.jsx here from lines 17-21](https://github.com/Ada-C22/flasky-front-end/blob/db8e1afd6d2e4cadf627841adcb1baec55544857/src/components/Cat.jsx#L17-L21) to see how the passed down event handler gets used.